packages/create-remix/templates/fly-stack/README.md

```diff
@@ -0,0 +1,85 @@
+# Remix Fly Stack
+
+- [Remix Docs](https://remix.run/docs)
+
+## Fly Setup
+
+1. [Install Fly](https://fly.io/docs/getting-started/installing-flyctl/)
+
+2. Sign up and log in to Fly
+
+```sh
+$ flyctl auth signup
```

Comment:
Get rid of the `$` from all these so people can use the built-in copy/paste stuff on github and/or just copy the whole block of scripts and execute them all at once.

![image](https://user-images.githubusercontent.com/1500684/151623781-e975a5b7-cdd5-4da7-82d5-85dcdfec05ad.png)

```suggestion
flyctl auth signup
```